rpm/xrootd-hdfs.spec

```diff
@@ -1,6 +1,6 @@
 Name: xrootd-hdfs
 Version: 2.1.4
-Release: 1%{?dist}
+Release: 2%{?dist}
```

Comment:
Instead of bumping the release, can you bump the verison to 2.1.5 (and update the changelog)?